speedy-links/src/speedy-links.ts

```diff
@@ -53,8 +53,16 @@ async function onPopState(): Promise<void> {
   await renderPage(location.href);
 }
 
-function isDifferentOriginUrl(url: string): boolean {
-  return new URL(url).origin !== location.origin;
+function isSameProjectURL(url: string): boolean {
+  const _url = new URL(url);
+  // Compare path prefixes, typically /doc/{projectId}
+  const currentRoot = location.pathname.slice(1).split('/');
+  const targetRoot = _url.pathname.slice(1).split('/');
+  return (
+    _url.origin === location.origin &&
+    currentRoot[0] === targetRoot[0] &&
+    currentRoot[1] === targetRoot[1]
+  );
```

Comment:
either I don't understand this logic, or it's not generic and as such should go into options, maybe a new option like `ignoreURL` or smth should be introduced

can you please elaborate in which project you encountered this issue and give an example of the links which should be loaded with speedy-links and should not in that project?

Reply:
ok, this was the problem 
![image](https://user-images.githubusercontent.com/137844/170049704-6297843f-08ea-48c1-8985-35ff8fd552f8.png)

and it's been fixed in https://github.com/divriots/dockit-core/pull/97